client/src/components/Common/Input/index.js

```diff
@@ -1,17 +1,27 @@
 import React from "react";
 import styled from "styled-components";
-import { spacing8 } from "../../../resources/style-constants";
+import { spacing16, theme1, grey1, grey2, spacing128, spacing8 } from "../../../resources/style-constants";
 
-const Wrapper = styled.div`
-  padding: ${spacing8} 0;
+const StyledInput = styled.input`
+    background-color: white;
+    margin: 0px;
+    padding: ${spacing16};
+    width: 40%;
```

Comment:
For this line, we want to be able to set the width to 100% if the boolean fullwidth is true. 
You can use props in order to achieve this. 
width: ${ ({fullwidth}) => fullwidth ? "100%" : "40%"}

Reply:
alright awesome